keyboards/vagrant_10/readme.md

```diff
@@ -0,0 +1,17 @@
+# Vagrant 10
+
+![Planck](https://raw.githubusercontent.com/Sho-Keebs/Vagrant-10/master/doc/vag.jpg)
+
+This is (yet another) simple and small macropad.
+
+* Keyboard Maintainer: [Shanduur](https://github.com/Shanduur)
+* Hardware Supported: Vagrant-10 PCB, Arduino Pro Micro
+* Hardware Availability: Groupbuys, Pro Micro can be found on [Aliexpress](https://www.aliexpress.com/wholesale?SearchText=arduino+pro+micro)
+
+Make example for this keyboard (after setting up your build environment):
+
+```bash
+qmk compile -kb vagrant-10 -km default 
+```
```

Comment:
```suggestion
    make vagrant_10:default

Flashing example for this keyboard:

    make vagrant_10:default:flash
```

Reply:
I don't really agree with that, as Docs states QMK bootstrap is the default way to install dependencies and compile keyboard binaries, but ehh, let it be.

Reply:
Currently we prefer to have new keyboard submissions follow the template as much as possible, however, yes, we will probably change these to use the CLI command as it gains adoption and functionality.